packs/excel/config.schema.yaml

```diff
@@ -0,0 +1,30 @@
+---
+  excel_file: 
```

Comment:
One option would be to make this a parameter of each action.

This way user doesn't have to update config file, etc. if they are working with multiple spreadsheets. A good middle ground between a config parameter and action parameter would be to make action parameter optional and use config file if action parameter is not provided, otherwise use the parameter (this way we would support both approaches and it would be easy to work with multiple spreadsheets).